GoogleForCreators/web-stories-wp
# Story Editor: Design Updates to Panel

## Summary
This PR implements the style updates in this ticket so the panes in the editor are closer to the designs. 

Also updated margin on panel captions & updated preset font weights to match figma designs here:
https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=2%3A31374

## Relevant Technical Choices
Just style updates

## To-do
I can take care of updating the accent color to purple here if it's not already documented in another ticket

## User-facing changes
Updated spacing in the editor panels

## Testing Instructions
Open up the editor and see that the design panels spacing between it's contents reflects the designs in figma better

---



Fixes #4851
